lib/livebook_web/live/hub/edit_live.ex

```diff
@@ -70,6 +70,8 @@ defmodule LivebookWeb.Hub.EditLive do
               hub={@hub}
               id="enterprise-form"
```

Comment:
We can delete the "enterprise" and "fly" branches from here, right?

Reply:
Or we do that in another PR. I am only saying so because we already deleted the new components, so might as well get rid of the edit ones. :)

Reply:
I'll add the join flow, so I'll delete the rest of the old stuff (fly and enterprise)